one/src/lib.rs

```diff
@@ -167,6 +169,24 @@ struct DaemonOpts {
             env = "CERAMIC_ONE_CORS_ALLOW_ORIGINS"
         )]
     cors_allow_origins: Vec<String>,
+
+    /// Enable experimental feature flags
+    #[arg(
+        long,
+        value_parser = parse_flags::<ExperimentalFeatureFlags>,
+        env = "CERAMIC_ONE_EXPERIMENTAL_FEATURE_FLAGS",
+        value_names = ["Authentication", "None"]
+    )]
+    experiemental_feature_flags: Vec<ExperimentalFeatureFlags>,
+
+    /// Enable feature flags
+    #[arg(
+        long,
+        value_parser = parse_flags::<FeatureFlags>,
+        env = "CERAMIC_ONE_FEATURE_FLAGS",
+        value_names = ["None"]
+    )]
+    feature_flags: Vec<FeatureFlags>,
```

Comment:
Do we need this one right now? I don't really like having `none` as an option for either of these. It seems like it should just be a vec and if it's empty it means none. having 3 features and none "enabled" is confusing.